grace/models/classifier.py

```diff
@@ -11,8 +11,9 @@ class Classifier(torch.nn.Module):
 
     def __init__(self) -> None:
         self.models = {
-            # "LINEAR" : Linear,
-            "GCN": GCN,
+            # "LINEAR" : LinearModel,
```

Comment:
this comments mean the the other two are meant to be implemented?

Reply:
Exactly, it's a placeholder for other options. As soon as I'm done with edge property calculation, I can extend the model options to other architectures (I'm unsure how to design this properly tho, to avoid unnecessary code duplication).